source/adapters/level_zero/image.cpp

```diff
@@ -759,12 +759,13 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesSampledImageCreateExp(
 }
 
 UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesImageCopyExp(
-    ur_queue_handle_t hQueue, void *pDst, void *pSrc,
+    ur_queue_handle_t UrQueue, void *pDst, void *pSrc,
```

Comment:
The rest of the params in this file follow the conventions of

1. camelCase, not PascalCase
2. Prefixes `p` for pointers, `h` for handles.

I'd prefer if this could remain compliant with these conventions, even if it meant changing more LOC.

I don't mind beyond that what naming you want in the various queue types, e.g.
```
auto hLegacyQueue = Legacy(hQueue);
```
or anything else.